tests/providers/databricks/hooks/test_databricks.py

```diff
@@ -158,6 +162,11 @@ def repair_run_endpoint(host):
     """
     return f"https://{host}/api/2.1/jobs/runs/repair"
 
+def get_cluster_endpoint(host):
+    """
+    Utility function to generate the get run endpoint given the host.
+    """
+    return f"https://{host}/api/2.0/clusters/get"
```

Comment:
why not utilize the endpoint you defined in the hook?

Reply:
I think the author wants to test whether the hook is returning the value correctly.

Reply:
I just follow the pattern of exiting codes in the file 🙏